test/plugin/scenarios/spring-3.1.x-scenario/src/main/webapp/WEB-INF/web.xml

```diff
@@ -23,6 +23,20 @@
          version="3.1">
     <display-name>skywalking-spring-3.1.x-scenario</display-name>
 
+    <filter>
+        <filter-name>setCharacterEncodingFilter</filter-name>
+        <filter-class>org.apache.catalina.filters.SetCharacterEncodingFilter</filter-class>
+        <init-param>
+            <param-name>encoding</param-name>
+            <param-value>GBK</param-value>
+        </init-param>
+    </filter>
+
+    <filter-mapping>
+        <filter-name>setCharacterEncodingFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
+
```

Comment:
@wu-sheng Here I meant

Reply:
Got it. Do we need to revert back? I don't want the GBK is a good example :)

Reply:
> Got it. Do we need to revert back? I don't want the GBK is a good example :)

I’m ok with both situations, will revert it